### PR TITLE
 fix UT that missed in PR 1062

### PR DESCRIPTION
**Pull request by yb01**
**What type of PR is this?**
> /kind bug

**What this PR does / why we need it**:
add missing UT changes in PR 1062

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: plugin/pkg/auth/authorizer/rbac/bootstrappolicy/testdata/cluster-roles.yaml [hunk 1]
```diff
@@ -642,6 +642,29 @@ items:
...
+    - deployments
+    - deployments/rollback
+    - deployments/scale
+    - deployments/status
+    - replicasets
+    - replicasets/scale
+    - replicasets/status
+    - statefulsets
+    - statefulsets/scale
+    verbs:
```
Comment (h-w-chen) [inline plugin/pkg/auth/authorizer/rbac/bootstrappolicy/testdata/cluster-roles.yaml:659]: it seems all verbs included. is there a wildcard for that purpose?

**Review by h-w-chen — APPROVED** [review]
/lgtm

File: plugin/pkg/auth/authorizer/rbac/bootstrappolicy/testdata/cluster-roles.yaml (hunk 1 above)
Comment (sonyafenge) [reply plugin/pkg/auth/authorizer/rbac/bootstrappolicy/testdata/cluster-roles.yaml:659]: kube-controller-manager need all read-write access for apps group, that's why all verbs included.

**Review by sonyafenge — APPROVED** [review]
/LGTM

**Pull request merged**